src/layer/tether.js

```diff
@@ -43,7 +43,14 @@ class Tether extends React.Component<TetherPropsT, TetherStateT> {
         this.popper && this.popper.scheduleUpdate();
       }
       if (this.state.isMounted !== prevState.isMounted) {
-        this.initializePopper();
+        if (!this.props.anchorRef) {
+          // eslint-disable-next-line no-console
+          console.warn(
+            `[baseui][TetherBehavior] ref has not been passed to the Popper's anchor element`,
```

Comment:
this lgtm. would it be helpful to provide a link to docs + example describing the problem?

Reply:
i also wonder if this should be limited to `__DEV__`

Reply:
Good point! Yes, just `__DEV__`.